### PR TITLE
Add support for new MQTT in memory estimator tool

### PR DESCRIPTION
**Pull request by aggarw13**
Add support for new MQTT library in memory estimator tool

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tools/memory_estimator/memory_estimator.py [hunk 1]
```diff
@@ -20,36 +20,40 @@
 
 __FREERTOS_SRC_DIR__ = os.path.join('FreeRTOS', 'Source')
 __FREERTOS_PLUS_SRC_DIR__ = os.path.join('FreeRTOS-Plus', 'Source')
-__IOT_LIBS_DIR__ = os.path.join(__FREERTOS_PLUS_SRC_DIR__, 'FreeRTOS-IoT-Libraries-LTS-Beta1')
+__IOT_LIBS_BETA1_DIR__ = os.path.join(__FREERTOS_PLUS_SRC_DIR__, 'FreeRTOS-IoT-Libraries-LTS-Beta1')
+__IOT_LIBS_BETA2_DIR__ = os.path.join(__FREERTOS_PLUS_SRC_DIR__, 'FreeRTOS-IoT-Libraries-LTS-Beta2')
 
 __LIB_NAME_TO_SRC_DIRS_MAPPING__ = {
     'light-mqtt' : [
```
Comment (aggarg) [inline tools/memory_estimator/memory_estimator.py:27]: Do we want `light-mqtt-beta1` and `light-mqtt-beta2`?

File: tools/memory_estimator/template/report_libs.json [hunk 2]
```diff
@@ -13,9 +13,13 @@
             "size_description":"Code Size of Lightweight MQTT API (example generated with GCC for ARM Cortex-M)"
         },
         {
-            "lib_name":"mqtt",
+            "lib_name":"mqtt-beta1",
             "size_description":"Code Size of MQTT (example generated with GCC for ARM Cortex-M)"
```
Comment (aggarg) [inline tools/memory_estimator/template/report_libs.json:17]: Update description to say Beta1?

File: tools/memory_estimator/template/report_libs.json [hunk 3]
```diff
@@ -13,9 +13,13 @@
...
         },
+        {
+            "lib_name":"mqtt-beta1",
```
Comment (aggarg) [inline tools/memory_estimator/template/report_libs.json:20]: beta2

File: tools/memory_estimator/template/report_libs.json (hunk 2 above)
Comment (aggarw13) [reply tools/memory_estimator/template/report_libs.json]: Updated

File: tools/memory_estimator/template/report_libs.json (hunk 3 above)
Comment (aggarw13) [reply tools/memory_estimator/template/report_libs.json]: Oops, fixed

File: tools/memory_estimator/memory_estimator.py (hunk 1 above)
Comment (aggarw13) [reply tools/memory_estimator/memory_estimator.py]: As we do not plan to have an equivalent FreeRTOS.org page for the Beta2 version of Lightweight MQTT under FreeRTOS Labs as the Beta1 version (existing page: https://freertos.org/mqtt/preconfiguredexamplesLW.html), we wouldn't need the `light-mqtt-beta2` entry..

**Pull request merged**